Android/app/src/go/intra/split/retrier.go

```diff
@@ -62,8 +62,8 @@ type retrier struct {
 	// Flag indicating when retry is finished or unnecessary.
 	retryCompleteFlag chan struct{}
 	// Flags indicating whether the caller has called CloseRead and CloseWrite.
-	readCloseFlag  chan struct{}
-	writeCloseFlag chan struct{}
+	readCloseFlag  atomic.Bool
```

Comment:
Please `import "atomic"` package. (see the build failure below)

Reply:
yikes!